code/api/src/migrations/1-user.js

```diff
@@ -26,6 +26,10 @@ module.exports = {
       updatedAt: {
         allowNull: false,
         type: Sequelize.DATE
+      },
+      //Add style
+      style: {
+        type: Sequelize.INTEGER
```

Comment:
Rather than altering a migration file that exits, we should create a new migration file to add an attribute/column to the table.